src/uu/free/src/free.rs

```diff
@@ -63,6 +76,14 @@ fn parse_meminfo() -> Result<MemInfo, Error> {
                 "SwapTotal" => mem_info.swap_total = parsed_value,
                 "SwapFree" => mem_info.swap_free = parsed_value,
                 "SReclaimable" => mem_info.reclaimable = parsed_value,
+                "LowTotal" => mem_info.low_total = parsed_value,
+                "LowUsed" => mem_info.low_used = parsed_value,
+                "LowFree" => mem_info.low_free = parsed_value,
+                "HighTotal" => mem_info.high_total = parsed_value,
+                "HighUsed" => mem_info.high_used = parsed_value,
```

Comment:
`LowUsed` and `HighUsed` are not keys in `/proc/meminfo`, see https://www.kernel.org/doc/html/latest/filesystems/proc.html#meminfo